Source/WebInspectorUI/UserInterface/Controllers/NetworkManager.js

```diff
@@ -229,6 +230,8 @@ WI.NetworkManager = class NetworkManager extends WI.Object
     get mainFrame() { return this._mainFrame; }
     get localResourceOverrides() { return this._localResourceOverrides; }
     get bootstrapScript() { return this._bootstrapScript; }
+    get failedSourceMapURLs() { return this._failedSourceMapURLs; }
+    get downloadingSourceMaps() { return this._downloadingSourceMaps; }
```

Comment:
it doesnt seem like these are actually used anywhere.  if so, please remove

Reply:
Added the `FIXEME`, removed the newline, changed `currentSession` to `currentTopLevelSession`, and removed the unnecessary getters in [7a17be1](https://github.com/WebKit/WebKit/pull/18733/commits/7a17be172e16a7e1cf40600f855ef61afda6465f).

Reply:
The getters were leftovers from an earlier exploration and didn't get cleaned up. Thanks for catching those!